control/jet_vane_model.cc

```diff
@@ -0,0 +1,69 @@
+#include "control/jet_vane_model.hh"
+
+#include "third_party/experiments/eigen_helpers.hh"
+
+namespace jet {
+namespace control {
+
+namespace detail {
+SE3 compute_T_vane_unit_from_aerodynamic_frame() {
+  const jcc::Vec3 vane_center_of_pressure_vane_unit_frame = jcc::Vec3(0.0, -0.030, 0.0);
```

Comment:
Make -.030 config? At least explain source of value.

Reply:
This is extremely reasonable; the goal is to load this from the CAD directly, so I'll do that in a follow-up diff since it's somewhat nontrivial